next/components/search/search-multicheckbox-facet.tsx

```diff
@@ -52,18 +52,21 @@ export function MultiCheckboxFacet({
                   )}`}
                   checked={checked}
                   onClick={() => (checked ? onRemove(value) : onSelect(value))}
+                  aria-label={`example_facet_${label}${getFilterValueDisplay(
+                    option.value
+                  )}-title`}
                 />
-                <label
+                <span
                   className="ml-2 text-sm text-steelgray"
-                  htmlFor={`example_facet_${label}${getFilterValueDisplay(
+                  title={`${t("filter-by")} ${getFilterValueDisplay(
                     option.value
-                  )}`}
+                  )} (${option.count.toLocaleString("en")})`}
                 >
                   {getFilterValueDisplay(option.value)}{" "}
                   <span className="text-steelgray">
                     ({option.count.toLocaleString("en")})
                   </span>
-                </label>
+                </span>
               </div>
             </li>
```

Comment:
I'm not sure about these aria-labels

For the one on line 55, voiceover is announcing it literally like this:

<img width="628" alt="CleanShot 2023-05-17 at 15 03 13@2x" src="https://github.com/wunderio/next4drupal-project/assets/22575651/b71c425b-7591-480a-944d-05d6ae6877b2">

And then for the span after the checkbox, don't we want it to remain as a `label`? 

I don't think we need to overcomplicate this. The {label} just before the list of checkboxes clearly says what the filter is ("Tags", "Content type"), so I feel like we're good as long as 1. the label for each checkbox and checked/unchecked status is clearly announced, and 2. the label and checkbox are associated with each other such that clicking the label also toggles its checkbox.